### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by kambereBr**
### In this pull request, as a team we:

1.  Used basic JavaScript syntax and DOM manipulations.
2. Used JavaScript arrays and objects to store and access all of the information for all projects cards.
3. Used JavaScript events to popup and close the window.
4. Made the popup responsive both for mobile and desktop versions.

### CODE REVIEW

**Review by Strangeal — COMMENTED** [review]
Hi @kambereBr, @Arslan49sp  and @mwovi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, that does have `ESLint` setup in your linter workflows. I would recommend adding to your workflow using the [Microverse linter config](https://github.com/microverseinc/linters-config) `ESLint` rules as it is designed to assist in writing code that aligns with industry standards and best practices. Not adding `ESLint` rules can lead to JavaScript code that may not adhere to industry standards, potentially resulting in code quality and maintainability issues. Instead, I recommended addressing any specific `ESLint` rule violations by finding alternative solutions to ensure that your code remains consistent and of high quality after adding it.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by kambereBr** [comment]
Thanks @Strangeal

**Comment by Strangeal** [comment]
@kambereBr You're welcome

File: assets/js/script.js [hunk 1]
```diff
@@ -33,4 +33,219 @@ for (let i = 0; i < linkMenu.length; i += 1) {
...
+  items: [
+    'Uber',
+    'assets/images/Counter.svg',
+    'Lead Developer',
+    'assets/images/Counter.svg',
+    '2018',
+  ],
+  liveVersionLink: 'https://kamberebr.github.io/Portfolio/',
+  sourceLink: 'https://github.com/kambereBr/Portfolio.git',
+};
```
Comment (Lucas-Masaba) [inline assets/js/script.js:102]: - [ ] As mentioned in the project requirements, this should be an array of objects to store JavaScript objects containing the information for each project. Every time you want to display a project on your portfolio, all you have to do is just add an object to this array. For example: 
```
const projects = [ {
  name: 'Uber navigation',
  description:
    'A smart assistant to make driving more safe, efficient and fun by unlocking your most expensive computer, your car.',
  image: 'assets/images/Snapshoot Portfolio.png',
  technologies: ['html', 'Ruby on rails', 'css', 'javascript'],
  items: [
    'Uber',
    'assets/images/Counter.svg',
    'Lead Developer',
    'assets/images/Counter.svg',
    '2018',
  ],
  liveVersionLink: 'https://kamberebr.github.io/Portfolio/',
  sourceLink: 'https://github.com/kambereBr/Portfolio.git',
},
{
  name: 'Facebook 360',
  description:
    "Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and video on Gear VR.",
  image: 'assets/images/Snapshoot Portfolio-4.png',
  technologies: ['html', 'Ruby on rails', 'css', 'javascript'],
  items: [
    'FACEBOOK',
    'assets/images/Counter.svg',
    'Full Stack Dev',
    'assets/images/Counter.svg',
    '2015',
  ],
  liveVersionLink: 'https://kamberebr.github.io/Portfolio/',
  sourceLink: 'https://github.com/kambereBr/Portfolio.git',
}]
```

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @kambereBr & @Strangeal  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Required linter checks are passing :heavy_check_mark: 
- UI is looking great :+1: 


## Required Changes ♻️
- [ ] In your `Getting Started` and `Future Features` sections in your README, I noticed that its talking about Ruby on Rails and yet this is an HTML, CSS and Javascript project. Please adjust both those sections to fit the current project you are working on which is the Portfolio. You should always keep in mind that your documentation should have the right information so as not to confuse the user.

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/js/script.js (hunk 1 above)
Comment (kambereBr) [reply assets/js/script.js]: Thanks, @Lucas-Masaba for your review!

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @kambereBr , @Arslan49sp and @mwovi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [ ] Nice job fixing the array of objects :rocket: 


 ## Required Changes ♻️
_Check the comments under the review._
- [ ] It would be better if we address the comment about the README file as mentioned in the previous review :+1: 
Screenshot from 2023-06-23 00-28-27
Screenshot from 2023-06-23 00-29-00
Let us customize the README file to reflect the current project

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kambereBr** [comment]
Thanks @addod19, now the README file is updated.

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @kambereBr 🙋‍♀️

Great work implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Your README looks good, well done!

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kambereBr** [comment]
Thanks @KanzaTahreem

**Pull request merged**